webapp/src/components/PublishPage/PublishPage.js

```diff
@@ -47,10 +59,27 @@ export default class PublishPage extends React.PureComponent {
                     isTxIdle={isTxIdle}
                     onPublish={onPublish}
                     onCancel={onCancel}
+                    isDisabled={!isLandAuthorized}
                   />
                   <TxStatus.Parcel parcel={parcel} />
                 </Grid.Column>
               </Container>
+              <br />
+              {!isLandAuthorized ? (
+                <Container text>
+                  <Grid.Column>
+                    <Message warning>
+                      <h3>
+                        <strong>Unauthorized</strong>
```

Comment:
I _think_ you can get a similar effect using the `header` prop on `Message`, if you want to give it a try